native-dnn/src/main/java/com/intel/analytics/bigdl/mkl/hardware/Affinity.java

```diff
@@ -14,8 +17,8 @@
         IMPL = new LinuxAffinity(CpuInfo.INSTANCE.getLogicalProcessorCount());
     }
 
-    public static int setAffinity() {
-        return IMPL.setAffinity();
+    public static void setAffinity() {
```

Comment:
why not returning the result ? how we can know if the binding works or not ?

Reply:
If not, the result will throw an exception.